include/Core/UI/Page/LogListPage.php

```diff
@@ -206,16 +206,21 @@ public function load_view_logs_assets( $hook ) {
 			return;
 		}
 
+		global $wp_scripts;
 		$email_log      = email_log();
 		$plugin_dir_url = plugin_dir_url( $email_log->get_plugin_file() );
 
 		wp_enqueue_style( 'jquery-ui-css', $plugin_dir_url . 'assets/vendor/jquery-ui/themes/base/jquery-ui.min.css', array(), '1.12.1' );
 		wp_enqueue_style( 'el-view-logs-css', $plugin_dir_url . 'assets/css/admin/view-logs.css', array( 'jquery-ui-css' ), $email_log->get_version() );
 
+		$ui  = $wp_scripts->query( 'jquery-ui-core' );
+		$url = "//ajax.googleapis.com/ajax/libs/jqueryui/{$ui->ver}/themes/smoothness/jquery-ui.css";
+		wp_enqueue_style( 'jquery-ui-smoothness', $url, array(), $ui->ver );
```

Comment:
We should pull it from the vendor directory.

WordPress plugin guidelines doesn't allow us to load the CSS/JS files directly from CDN domains.

Reply:
> We should pull it from the vendor directory.

Okay, got it.

> WordPress plugin guidelines doesn't allow us to load the CSS/JS files directly from CDN domains.

The above code is used in Bulk Delete. We should get that changed too. I'll add a GitHub issue.

Reply:
@sudar,

I've opened an issue in GitHub https://github.com/sudar/bulk-delete/issues/582

Reply:
@mariadanieldeepak 

Yeah, we need to change it in Bulk Delete as well.

Let me know once you have updated this PR.

Reply:
@sudar,

> Let me know once you have updated this PR.

I've resolved this now. Also, the Codacy failure is expected, in this case.